types/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// TypeScript Version: 3.0
+
+import {Transformer} from 'unified'
+
+declare namespace strip {
+  interface Options {
+    /**
+     * List of node types to leave unchanged.
+     */
+    keep?: Array<
+      | 'heading'
+      | 'text'
+      | 'inlineCode'
+      | 'image'
+      | 'imageReference'
+      | 'break'
+      | 'blockquote'
+      | 'list'
+      | 'listItem'
+      | 'strong'
+      | 'emphasis'
+      | 'delete'
+      | 'link'
+      | 'linkReference'
+      | 'code'
+      | 'horizontalRule'
+      | 'thematicBreak'
+      | 'html'
+      | 'table'
+      | 'tableCell'
+      | 'definition'
+      | 'yaml'
+      | 'toml'
```

Comment:
could this a) be imported from `@types/mdast`, or b) maybe even be dropped, as users might want to keep custom node types

Reply:
i took the list from https://github.com/remarkjs/strip-markdown/blob/main/index.js#L7-L33 but i guess it would be fine to just make this `string[]`

Reply:
oh wait, I guess it makes sense because there are errors: https://github.com/remarkjs/strip-markdown/blob/ba891bdde79fff0efd32e7d18fc7ea4f83014965/index.js#L53